server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestNodesInfoAction.java

```diff
@@ -24,6 +26,7 @@
 
 import static org.elasticsearch.rest.RestRequest.Method.GET;
 
+@ServerlessScope(Scope.PUBLIC)
```

Comment:
Change to `INTERNAL`, which will let Kibana access these when the expected header is utilized